package.json

```
@@ -3,7 +3,7 @@
   "type": "module",
   "private": true,
   "engines": {
-    "node": ">=18.16"
+    "node": ">=20"
```

change this back so we can deploy still

```suggestion
    "node": ">=18.16"
```